src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -95,8 +97,8 @@ <h4 class="h4" id="links">{% trans "Links" %}</h4>
         {% render_column start=related_products_start span=4 compact=True %}
             {# Related products. #}
             {% if object.related_products.published.exists %}
-                <nav class="link-list">
-                    <h4 class="h4" id="see">{% trans "Zie ook" %}</h4>
+                <nav class="link-list" aria-label="{% trans "Zie ook links" %}">
```

Comment:
I thiink "gerelateerd links" is a bit more descriptive.

Reply:
Renamed aria-label